R2API/ContentManagment/R2APIContentManager.cs

```diff
@@ -12,8 +12,9 @@
 using UnityEngine;
 using UnityEngine.Networking;
 using Object = UnityEngine.Object;
+using R2API.MiscHelpers;
 
-namespace R2API {
+namespace R2API.ContentManagment {
```

Comment:
management

Reply:
I hate english with their silent letters

Reply:
I'm partial to Mangement (i've done that before)